templates/nginx.conf.j2

```diff
@@ -80,3 +81,12 @@ server {
         deny all;
     }
 }
+
+{% if laravel_https_redirect -%}
+server {
+    listen         80;
+    listen         [::]:80;
```

Comment:
? This is for ipv6, not sure it's required. We already have listen on port 80 above. Should be enough.